packages/tools/src/eventDispatchers/annotationInterpolationEventDispatcher.ts

```diff
@@ -0,0 +1,38 @@
+import { eventTarget } from '@cornerstonejs/core';
+import Events from '../enums/Events';
+import InterpolationManager from '../utilities/contourROITool/InterpolationManager';
+
+const enable = function (element: HTMLDivElement) {
+  eventTarget.addEventListener(
+    Events.ANNOTATION_LABEL_CHANGE,
+    InterpolationManager.handleAnnotationLabelChange as EventListener
+  );
```

Comment:
I believe you are pushing for annotations and labels as being segmentations. However, we are advocating for another method that involves contour segmentation representation, which will serve as the foundation for grouping contours (being able to color, fill, etc. etc.). The PR (https://github.com/cornerstonejs/cornerstone3D/pull/983) will be merged shortly, so it might be a good idea to incorporate that concept into your PR. Let's definitely schedule a meeting to discuss this further. I believe email communications have already begun to arrange a suitable time.